apps/web/components/imports/schema/ColumnsTable.tsx

```diff
@@ -14,20 +14,40 @@ import { EditIcon } from '@assets/icons/Edit.icon';
 import { CloseIcon } from '@assets/icons/Close.icon';
 import { CheckIcon } from '@assets/icons/Check.icon';
 import { DeleteIcon } from '@assets/icons/Delete.icon';
+import { MultiSelect } from '@mantine/core';
 
 interface ColumnsTableProps {
   templateId: string;
 }
 
 export function ColumnsTable({ templateId }: ColumnsTableProps) {
   const [showAddRow, setShowAddRow] = useState(false);
-  const { register, columns, control, handleSubmit, onEditColumnClick, onDeleteColumnClick, isColumnCreateLoading } =
-    useSchema({
-      templateId,
-    });
+  const [isFocuedOnSelect, setIsFocusOnSelect] = useState(false);
```

Comment:
Spelling typo in `isFocuedOnSelect` and `setIsFocusOnSelect`. May be `isSelectFocused` and `setIsSelectFocused` naming will seems good.

Reply:
have corrected the naming conviction